consensus/src/consensus/remote_event_dispatcher.rs

```diff
@@ -213,6 +215,25 @@ impl<N: Network> RemoteEventDispatcher<N> {
             network_event_rx: network_events,
         }
     }
+
+    // This is a helper function to determine if we need to create notifications
```

Comment:
```suggestion
    /// This is a helper function to determine if we need to create notifications
```
Not strictly necessary but since it is already a comment on the function, I'd suggest to add it